hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/TestDataUtil.java

```diff
@@ -63,7 +61,8 @@ public static OzoneBucket createVolumeAndBucket(MiniOzoneCluster cluster,
     if (bucketLayout != null) {
       builder.setBucketLayout(bucketLayout);
     }
-    omBucketArgs = builder.build();
+    omBucketArgs =
+            builder.setDefaultReplicationConfig(new DefaultReplicationConfig(new ECReplicationConfig(3,2))).build();
```

Comment:
Unrelated, but it would be nice to have ECReplicationConfig objects as singletons.

I suspect we'll end up seeing a lot of duplicate ECReplicationConfig objects in the heap in the future.